base/android/jni_android.cc

```diff
@@ -30,9 +33,52 @@ ABSL_CONST_INIT thread_local void* stack_frame_pointer = nullptr;
 
 bool g_fatal_exception_occurred = false;
 
+/* Cobalt specific hack to move Java classes to a custom namespace.
```

Comment:
fwiw these could be made static inline functions in a cobalt-owned helpers file instead, but I'll leave that to your discretion